core/interfaces/i_toolbox_item.js

```diff
@@ -113,6 +113,15 @@ Blockly.ISelectableToolboxItem.prototype.getContents;
  */
 Blockly.ISelectableToolboxItem.prototype.setSelected;
 
+/**
+ * Gets the html element that is clickable.
+ * The parent toolbox element receives clicks. This method makes it possible for the parent toolbox
+ * to add an id to the this element and pass the event to this items onClick method.
```

Comment:
I was a bit confused by this regardless... I think it might be more clear (if i'm understanding correctly) to remove the "makes it possible" part and say something like
"The parent toolbox will add an id to this element so it can pass the onClick event to the correct toolboxItem." is that accurate?

Reply:
Yup! That makes a lot more sense. Thanks!